cms/djangoapps/contentstore/utils.py

```diff
@@ -1847,7 +1872,10 @@ def get_container_handler_context(request, usage_key, course, xblock):  # pylint
         create_xblock_info,
     )
     from openedx.core.djangoapps.content_staging import api as content_staging_api
+    from cms.djangoapps.contentstore.views.course import _course_outline_json
 
+    course_structure = _course_outline_json(request, course)
```

Comment:
[nit] I think we need to move computing of `sequence_ids` to `cms/djangoapps/contentstore/rest_api/v1/views/vertical_block.py`
We don't need to have `sequence_ids` in the shared utils context.

Reply:
I would keep it inside the util function in order to have data collection incapsulated